File: ProgrammingFundamentals/C#/03.DataTypesAndVariables/Exercises/P16_ComparingFloats/P16_ComparingFloats.cs
using System;

namespace P16_ComparingFloats
{
    class P16_ComparingFloats
    {
        static void Main(string[] args)
        {
            double firstNum = double.Parse(Console.ReadLine());
            double secondNum = double.Parse(Console.ReadLine());
            double epsilon = 0.000001;

            Console.WriteLine(Math.Abs(firstNum - secondNum) < epsilon);
        }
    }
}

File: ProgrammingFundamentals/Java/03.DataTypesAndVariables/Exercises/src/P16_ComparingFloats.java
import java.util.Scanner;
public class P16_ComparingFloats {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        double firstNum = Double.parseDouble(console.nextLine());
        double secondNum = Double.parseDouble(console.nextLine());
        double epsilon = 0.000001;
        System.out.println((Math.abs(firstNum - secondNum) < epsilon) ? "True" : "False");
    }
}
